storage/queue.go

```diff
@@ -131,12 +138,31 @@ func (bq *baseQueue) Length() int {
 	return bq.priorityQ.Len()
 }
 
+// SetDisabled turns queue processing off or on as directed.
+func (bq *baseQueue) SetDisabled(disabled bool) {
+	if disabled {
+		atomic.StoreInt32(&bq.disabled, 1)
+	} else {
+		atomic.StoreInt32(&bq.disabled, 0)
+	}
+}
+
 // Start launches a goroutine to process entries in the queue. The
 // provided stopper is used to finish processing.
 func (bq *baseQueue) Start(clock *hlc.Clock, stopper *stop.Stopper) {
 	bq.processLoop(clock, stopper)
 }
 
+// Add adds the specified range to the queue, regardless of the return
+// value of bq.shouldQueue. The range is added with specified
+// priority. If the queue is too full, the range may not be
+// added. Returns an error if the range was not added.
```

Comment:
the comment in `MaybeAdd` below stipulates that this would drop the lowest-priority item from the queue. Which one is it?